rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala

```diff
@@ -143,7 +146,9 @@ object ProcNormalizeMatcher {
       val desugaredIf = Match(
         targetResult.par,
         List((GBool(true), trueCaseBody.par), (GBool(false), falseCaseBody.par)),
-        freeCount)
+        freeCount,
+        trueCaseBody.par.locallyFree | falseCaseBody.par.locallyFree
```

Comment:
`targetResult.par.locallyFree | trueCaseBody.par.locallyFree | falseCaseBody.par.locallyFree`

Reply:
Good catch. Thank you.